react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -15,94 +16,82 @@ import { updateDialInNumbers } from '../actions';
 import { InfoDialog } from './info-dialog';
 
 /**
- * The amount of time, in milliseconds, to wait until automatically showing
- * the {@code InfoDialog}. This is essentially a hack as automatic showing
- * should happen in a lonely call and some time is needed to populate
- * participants already in the call.
+ * The type of the React {@code Component} props of {@link InfoDialogButton}.
  */
-const INFO_DIALOG_AUTO_SHOW_TIMEOUT = 1500;
+type Props = {
+
+    /**
+     * The redux state representing the dial-in numbers feature.
+     */
+    _dialIn: Object,
+
+    /**
+     * Whether or not the {@code InfoDialog} should display automatically when
+     * in a lonely call.
+     */
+    _disableAutoShow: boolean,
+
+    /**
+     * Whether or not the local participant has joined a
+     * {@code JitsiConference}. Used to trigger auto showing of the
+     * {@code InfoDialog}.
+     */
+    _isConferenceJoined: Boolean,
+
+    /**
+     * The URL for a currently active live broadcast
+     */
+    _liveStreamViewURL: string,
```

Comment:
I hate when other reviewers do this, but seems like I can't avoid it either. So I could have seen this earlier: can it be undefined? Maybe worth adding the ? to the type if yes.